### ISSUE TITLE
'prompt' config isn't working

### ISSUE THREAD

**Issue by phil-s** [bobthecow/psysh#755]
I'm trying to restore the prompt to `>>>` using the `'prompt'` config per https://github.com/bobthecow/psysh/wiki/Config-options but it's having no effect.  Other config changes (such as colour) are working, so this seems like a bug/regression.

**Comment by bobthecow** [comment]
You're right! We recently added support for [themes](https://github.com/bobthecow/psysh/wiki/Themes), which control the input, continuation and replay prompts, the return value indicator, formatting styles (colors, bold, etc) and whether or not to use compact output.

You can restore everything to the previous styles by using `'theme' => 'classic'`, or do some mix of old and new styles via custom themes [see the docs for more info](https://github.com/bobthecow/psysh/wiki/Themes).

This is _also_ an unintended regression. It should still support the old option as long as there's no theme set, so I'll get that fixed.

**Comment by phil-s** [comment]
Aha.  That covers all of my needs nicely.  Thank you!

**Comment by bobthecow** [comment]
And now it's in the latest release. Thanks for the bug report!
